src/tenant.ts

```diff
@@ -65,15 +65,16 @@ export class Tenant {
         );
     }
 
-    modifyTenant(passwd: string, extra?: object) {
+    modifyTenant(passwd: string, fabricName='_system', extra={}) {
```

Comment:
@dwivedianurag Please add `active` also in argument.

Reply:
made the required changes according to the updated  ticket